README.md

```diff
@@ -71,6 +75,23 @@ module.exports = {
 
 **📝 Note: It is important to have the plugin in the first position in order to report every other plugins.**
 
+### Esbuild
+
+Add the plugin to your esbuild configuration.
+
+```js
+const esbuild = require('esbuild');
+const { BuildPlugin } = require('@datadog/build-plugin/dist/esbuild');
+
+esbuild.build({
+  [...] // All your configuration needs
+  plugins: [
+    [...] // All your plugins
+    BuildPlugin()
```

Comment:
Position doesn't matter for this one (unlike Webpack where the plugin has to be last)?

Btw, does it have an overhead at build time?

Reply:
Yes, the position doesn't matter here because esbuild runs the plugins' setup once they all registered and not as soon as they instantiate.

The overhead at build time is as minimal as for webpack.